sfm_pc/base_views.py

```diff
@@ -143,3 +140,9 @@ def get_objects_to_update(self):
         Composition in question.
         """
         raise NotImplementedError('This method must be implemented on children.')
+
+    def get_cancel_url(self):
+        """
+        Return the route that the Cancel button should link to in the template.
+        """
+        raise NotImplementedError('This method must be implemented on children.')
```

Comment:
We can clean this method up since `BaseDeleteRelationshipView` inherits from `BaseDeleteView`.

Reply:
Cool. done